pkgs/top-level/default.nix

```diff
@@ -72,75 +57,9 @@ let
   # deterministically inferred the same way.
   nixpkgsFun = newArgs: import ./. (args // newArgs);
```

Comment:
There should be comments to explain this, but actually I think this is OK. `config` is fine to `include`, but platform actually shouldn't be included because this function is usually called with a non-native `system`, which causes a different `platform` to be inferred. It would be bad to force the native `system`'s inferred `platform`.